packages/tools/devtools/devtools-view/package.json

```diff
@@ -138,7 +161,35 @@
 					"api-extractor:commonjs"
 				],
 				"script": false
-			}
+			},
+			"build:test:jest:cjs": [
+				"^tsc",
```

Comment:
Should be able to remove this since ^tsc is a sub dependency (not sure what else to call it) through the utils.cjs task.